app/assets/javascripts/viewers.js

```diff
@@ -0,0 +1,9 @@
+var onReadyViewers = function() {
+  if (newOrEdit(["moments", "strategies"])) {
+  	$("#viewers_all").click(function() {
+  		$("#viewers_list :checkbox").prop("checked", $(this).prop("checked"));
```

Comment:
What happens when viewers > 20? will we have a long list of checkboxes?  - just something I noticed when I was playing with the branch